examples/elixir/get_started/06-secure-channel-over-two-uds-transport-hops-initiator.exs

```diff
@@ -0,0 +1,27 @@
+["setup.exs"] |> Enum.map(&Code.require_file/1)
+
+# Register this process as worker address "app".
+Ockam.Node.register_address("app")
+
+# Start the TCP Transport Add-on for Ockam Routing.
+Ockam.Transport.UDS.start()
+
+# Create a vault and an identity keypair.
+{:ok, vault} = Ockam.Vault.Software.init()
+{:ok, identity} = Ockam.Vault.secret_generate(vault, type: :curve25519)
+
+# Connect to a secure channel listener and perform a handshake.
+alias Ockam.Transport.UDSAddress
+r = [UDSAddress.new("/tmp/sock1"), "h1", UDSAddress.new("/tmp/sock2"), "secure_channel_listener"]
```

Comment:
We're trying to steer away from using implicit connections expressed in specific address types and towards explicit connection workers.
I think we can simplify example by first creating a connection worker like `{:ok, client} = UDS.Client.create("/tmp/sock1")` and then sending messages through it.
That removes support for multi-hop thing and simplifies the example as it would only have 2 nodes.

Reply:
Thanks @hairyhum, this is great feedback! I went back and did some more reading and implemented an `Ockam.Transport.UDS.Client` and used it in a Routing example, similar to how you specified it in your main comment. I still have to clean up the get_started examples and the UDSAddress.